sqlalchemy_ibmi/base.py

```diff
@@ -676,88 +756,374 @@ class DB2Dialect(default.DefaultDialect):
+                            Column("FKCOLUMN_NAME", CoerceUnicode, key="fkcolname"),
+                            Column("PK_NAME", CoerceUnicode, key="pkname"),
+                            Column("PKTABLE_SCHEM", CoerceUnicode, key="pktabschema"),
+                            Column("PKTABLE_NAME", CoerceUnicode, key="pktabname"),
+                            Column("PKCOLUMN_NAME", CoerceUnicode, key="pkcolname"),
+                            Column("KEY_SEQ", sa_types.Integer, key="colno"),
+                            schema="SYSIBM")
+
+    sys_views = Table("SYSVIEWS", ischema,
+                      Column("TABLE_SCHEMA", CoerceUnicode, key="viewschema"),
+                      Column("TABLE_NAME", CoerceUnicode, key="viewname"),
+                      Column("VIEW_DEFINITION", CoerceUnicode, key="text"),
+                      schema="QSYS2")
+
+    sys_sequences = Table(
+        "SYSSEQUENCES",
+        ischema,
+        Column(
+            "SEQUENCE_SCHEMA",
+            CoerceUnicode,
+            key="seqschema"),
+        Column(
+            "SEQUENCE_NAME",
+            CoerceUnicode,
+            key="seqname"),
+        schema="QSYS2")
+
+    def get_temp_table_names(self, connection, schema=None, **kw):
+        pass
+
+    def get_temp_view_names(self, connection, schema=None, **kw):
+        pass
+
+    def get_check_constraints(self, connection, table_name, schema=None, **kw):
+        pass
+
+    def get_table_comment(self, connection, table_name, schema=None, **kw):
+        pass
+
+    def _get_server_version_info(self, connection):
+        pass
+
+    def do_begin_twophase(self, connection, xid):
+        pass
+
+    def do_prepare_twophase(self, connection, xid):
+        pass
+
+    def do_rollback_twophase(self, connection, xid, is_prepared=True, recover=False):
+        pass
+
+    def do_commit_twophase(self, connection, xid, is_prepared=True, recover=False):
+        pass
+
+    def do_recover_twophase(self, connection):
+        pass
+
+    def set_isolation_level(self, dbapi_conn, level):
+        pass
+
+    def get_isolation_level(self, dbapi_conn):
+        pass
 
-        self._reflector = self._reflector_cls(self)
 
     # reflection: these all defer to an BaseDB2Reflector
     # object which selects between DB2 and AS/400 schemas
     def initialize(self, connection):
         super(DB2Dialect, self).initialize(connection)
         self.dbms_ver = getattr(connection.connection, 'dbms_ver', None)
         self.dbms_name = getattr(connection.connection, 'dbms_name', None)
-        
-    def normalize_name(self, name):
-        return self._reflector.normalize_name(name)
-
-    def denormalize_name(self, name):
-        return self._reflector.denormalize_name(name)
 
     def _get_default_schema_name(self, connection):
-        return self._reflector._get_default_schema_name(connection)
+        """Return: current setting of the schema attribute"""
```

Comment:
This looks totally different. I thought this PR was just linter changes.

Reply:
I ran the linter after making the consolidation changes. Should I go through and lint on master even though those some of those files will be deleted in the consolidation PR.

Reply:
Yeah. It's a little more work, but the benefit is that by running the linter and formatter we can be relatively assured that the code being changed is just for linting, not any substantial changes.

I'd do it in two commits: one with the autoformatter run and one with manual adjustments. The autoformatter commit should contain the commands run to do the formatting as well.

Reply:
Let's also do enable full linting the PR in a separate PR.